### ISSUE TITLE
Http request Received still not showing in Impact analysis when connector is blocked

### ISSUE THREAD

**Issue by Cathi029** [microsoft/coe-starter-kit#9277]
> Hello @Jenefer-Monroe , I installed the November update, but unfortunately nothing changed in the impact analysis.
> ![Image](https://github.com/user-attachments/assets/9737be74-626a-43e0-ae67-ad56d8265e58)
> Power Platform Admin View App correctly shows the connector but the Impact Analysis is not showing the Flow under impacted when blocking it for this environment.
> 
> Kind regards
> Cathi 

 _Originally posted by @Cathi029 in [#9104](https://github.com/microsoft/coe-starter-kit/issues/9104#issuecomment-2470582630)_

**Comment by Jenefer-Monroe** [comment]
Did you run the setup wizard after upgrade? It will run the flow that updates these.

**Comment by Cathi029** [comment]
No, I did not. Which Flow should be triggered to have the update? But I also had a new installation in the meantime and I used the setup wizard for initial setup and we have the same problem.

**Comment by Jenefer-Monroe** [comment]
Yes you need to use the Setup Wizard after every upgrade. It will ensure any new env vars are filled, new flows are turned on, and periodic activities occur. More detail below. 
Please run through the setup wizard and then let me know if you still face the issue of the connector not appearing in the DLP app. 

## Use the setup wizard
Its highly recommended that you use the Setup Wizard to walk you through setup and upgrades. It will ensure all the new flows are turned on in the correct order, trigger things like population of the bookmarks screen and do other cleanup steps needed.
You should do this both for upgrades and clean installs.
To use it, first perform the import of the solution install or upgrade manually, then boot the app and walk through the screens.
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/b84cc1ff-383a-497c-9236-569e8313c73c)
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/ed02f0d0-c276-48cc-90af-47e5bf5a928e)

**Comment by Cathi029** [comment]
Hello, but for the ne installation I mentioned above, it should work as expected, right? "But I also had a new installation in the meantime and I used the setup wizard for initial setup and we have the same problem."
As mentioned I used the setup wizard as described in your message.

**Comment by Cathi029** [comment]
No, not the connector is missing, Impactet Flows are not showing up when blocking this specific connector:
I bult a flow ohny using the http request received connector, and blocked it for this environment. The Flow is not showing under impacted Flows but will be deaktivated when aktivating the DLP Policy.
![Image](https://github.com/user-attachments/assets/ade279b4-4087-4e0c-abd7-f803a8dbb664)
![Image](https://github.com/user-attachments/assets/36f8310d-2705-4659-b3be-cc88bdb55c8e)
![Image](https://github.com/user-attachments/assets/c3aa9c9f-2fa9-4995-8801-52f539c71d99)

**Comment by Jenefer-Monroe** [comment]
Yes unfortunately since we added the connector to the kit recently, objects will have to go through inventory before they have that noted. 

If you mark the flow as "inventory me" you should see it show the connector after the next inventory run. 

You can set the "inventory me" property here:
![Image](https://github.com/user-attachments/assets/6f640807-a6fa-412e-9759-01222c498e69)

And you can see if its been updated correctly the next day here: 
![Image](https://github.com/user-attachments/assets/fa7f5237-a636-4a1d-89f8-d0f8350ca5cd)

**Comment by Cathi029** [comment]
Hello, 
then I need to know the flows that are blocked in advance. Inventory me is empty or “no” for all flows. This makes little sense as I want to use the impact analysis to identify these flows in the first place. 
Does this procedure only apply to the Http received connector? What influence would it have if I set all flows to inventory me=yes?  Or which other functions do not work if the setting is not = yes?

Will there be an update here? At the moment I don't know how to explain this to my customers to be honest :)

Thanks for your support!

**Comment by Jenefer-Monroe** [comment]
I hear you. Unfortunately this is just a limitation on the low code architecture we are using here. 

The API hit to gather the tenant inventory is extraordinary and so we only look for recently updated objects for inventory, or objects that you set explicitly to inventory. 

The real solution is to have this information be delivered by the product team where they can use pro-code to get this information to you. 

For the kit, yes you can rerun the entire inventory to get this information by setting all flows to "inventory me" with the Excel Add-in. 

Sorry for the inconvenience.

**Comment by Cathi029** [comment]
okay, thank you. 
I created the flow with the http received trigger a few days ago. The kit detected it but created it with inventoryMe = no, so all new Apps and Flows will have this status and  before I want so see all Apps and Flows in the impact analysis, I need do to the Excel Workaround?
Which Features other than the impact analysis is not working for Apps and Flows with setting InventoryMe=no or blank?
This would be really important to know.

**Comment by Jenefer-Monroe** [comment]
InventoryMe is just for when you want to inventory outside the scope of the normal inventory. 
Its not something that we set, its something you would set to inventory out of the normal runs.

**Comment by Jenefer-Monroe** [comment]
Please share some screenshots here
1. the flow you made that you expect to see in the DLP app
2. the flows connections in the Power Platform Admin View app
3. the DLP app as you walk through the repro.

**Comment by Cathi029** [comment]
Hi, 

actually I have shared these Screenshots already two weeks ago. Here are some additionals Screenshots..
![Image](https://github.com/user-attachments/assets/7a2595f1-3ec8-4981-8f61-1bc9f4e0451b)
![Image](https://github.com/user-attachments/assets/a2be7fd0-7238-4db3-9806-67505ee1d10a)
![Image](https://github.com/user-attachments/assets/ec73501d-783e-41a6-8d69-383647fce1f9)
![Image](https://github.com/user-attachments/assets/4509c9e9-139b-4913-a9ce-70f8a85a9b23)
![Image](https://github.com/user-attachments/assets/0116d885-3c43-47cb-850e-33e1fe3e1ac8)
![Image](https://github.com/user-attachments/assets/30cce040-298f-473f-a2e1-0a380bf95faf)
![Image](https://github.com/user-attachments/assets/7c8a1340-7957-45df-96a8-d2846ad5922d)

**Comment by Jenefer-Monroe** [comment]
Looks like there is a mismatch between app and flow in how we call this connector.
See where to fix in the app I think but need more testing.

**Comment by AmarSaiRam** [comment]
Dear User, 

We noticed that this issue has been pending for quite some time, and we have not received any recent updates from your end. In the absence of further communication, we are proceeding to close this ticket, assuming that your concerns have been addressed or resolved. 
If you continue to experience any difficulties related to this matter, please feel free to open a new support ticket. This will help us ensure prompt and effective assistance, rather than reopening the new Issue. 
Thank you for your understanding and cooperation. 

Best Regards,
